cluster.go

```diff
@@ -1312,6 +1312,10 @@ func (c *ClusterClient) pipelineProcessCmds(
 	err = cn.WithReader(c.opt.ReadTimeout, func(rd *proto.Reader) error {
 		return c.pipelineReadCmds(node, rd, cmds, failedCmds)
 	})
+	if err == nil {
```

Comment:
If you return `err` anyway why don't you return it as it is?

Reply:
Fixed